README.md

```diff
@@ -626,9 +627,184 @@ class BookList extends React.Component {
 
 # Child nodes
 
-> TODO
-> ViewChild vs refs
-> ContentChild vs this.props.children
+### AngularJS
+In an [AngularJS component](https://docs.angularjs.org/guide/component), we have access to the child node by injecting [`$element`](https://docs.angularjs.org/api/ng/function/angular.element) to its controller.
+```js
+class TextInputCtrl {
+  constructor($scope, $element) {
+    // $element is passed to the constructor and stored in an instance variable.
```

Comment:
No need for the comment. DI was explained before. Just add ` 'ngInject';` as in other examples.